data/projects.yml

```
@@ -0,0 +1,16 @@
+title: Projects
```

We could make this more explicit, also looks nicer on the website

```suggestion
title: OpenRail Projects
```

---

Ok, then we should probably also change the title of the member section from "Membership" to "OpenRail Members".

---

Good idea, will do it separately